tests/state/test_particle.py

```diff
@@ -0,0 +1,25 @@
+from expertsystem.state.particle import DATABASE, create_antiparticle
+from expertsystem.ui import load_default_particle_list
+
+
+def test_create_antiparticle():
```

Comment:
I think a more solid test would be to use `pytest.mark.parametrize` and test several particles.

Reply:
Done